src/seedu/addressbook/AddressBook.java

```diff
@@ -486,13 +486,20 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
             final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            if (!Collections.disjoint(toUpperCase(wordsInName), toUpperCase(keywords))) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+    private static HashSet<String> toUpperCase(Collection<String> strings) {
+	 	HashSet<String> upperCaseStrings = new HashSet<>();
+	    for (String normalString : strings) {
+	     	upperCaseStrings.add(normalString.toUpperCase());
+	    }
+	    return upperCaseStrings;
+	 }
     /**
```

Comment:
When you modify the behavior, remember to update documentation and tests as well.